dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -44,30 +44,45 @@ export const NoExpiringRuns = () => {
 export const NewRunsHeading = () => {
   const [dropdownOpen, setDropdownOpen] = useState(false);
   const dispatch = useDispatch();
+  const { endpoints } = useSelector((state) => state.apiEndpoint);
 
   const actions = [
```

Comment:
Since this table is static, we don't need to construct it on each call to this function; it would be better if it were declared outside this function, before line 44.

Reply:
Done!